docs/tool_jdmpview.md

```diff
@@ -36,16 +36,20 @@ The dump viewer is useful for diagnosing `OutOfMemoryError` exceptions in Java&t
 
 ### Starting the dump viewer
 
-`jdmpview (-core <core file> | -zip <zip file>) [-notemp]`
+`jdmpview [-J<vm option>] (-core <core file> | -zip <zip file>) [-notemp]`
 
 
 |          Input option   |  Explanation                                                                                           |
 |-------------------------|--------------------------------------------------------------------------------------------------------|
 | `-core <core file>`     | Specifies a dump file.                                                                                 |
-| `-zip <zip file>`       | Specifies a compressed file containing the core file and associated XML file (produced by the [dump extractor (`jextract`)](tool_jextract.md) tool on AIX&reg;, Linux&reg;, and macOS&reg; systems). |
+| `-zip <zip file>`       | Specifies a compressed file containing the core file (produced by the [dump extractor (`jextract`)](tool_jextract.md) tool on AIX&reg;, Linux&reg;, and macOS&reg; systems). |
 | `-notemp`               | By default, when you specify a file by using the `-zip` option, the contents are extracted to a temporary directory before processing. Use the `-notemp` option to prevent this extraction step, and run all subsequent commands in memory. |
+| `-J-Dcom.ibm.j9ddr.path.mapping=<mappings>` | `<mappings>` is a `:` separated list of native library mappings, each of the form `old-path=new-path`. |
+| `-J-Dcom.ibm.j9ddr.library.path=<path>` | `<path>` is a `:` separated list of paths to search for native libraries. |
 
-<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** The `-core` option can be used with the `-zip` option to specify the core and XML files in the compressed file. Without these options, `jdmpview` shows multiple contexts, one for each source file that it identified in the compressed file.  
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** The `-core` option can be used with the `-zip` option to specify the core file in the compressed file. Without these options, `jdmpview` shows multiple contexts, one for each source file that it identified in the compressed file.
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** On AIX, Linux some `jdmpview` commands, such as those related to callsites, require locating the native libraries referenced by the core. See the list of native libraries via the `info mod` command when running `jdmpview` on a core. If the libraries are not in the original locations, `jdmpview` can locate them using the path mapping and library path options specified on the `jdmpview` command line.  
```

Comment:
I found this paragraph a little confusing when I first read it, so suggesting some slight rewording.

**Note:** On AIX and Linux, some `jdmpview` commands need to locate the native libraries that are referenced by the core.  For example, commands that relate to callsites. First, check the list of native libraries by running `jdmpview` on a core with the `info mod` parameter. If the libraries are not in the original locations, ensure that `jdmpview` can locate them by specifying the  path mapping (`-J-Dcom.ibm.j9ddr.path.mapping`) and library path (`-J-Dcom.ibm.j9ddr.library.path`) options on the `jdmpview` command line.

Reply:
I also noticed that `info mod` is not listed in the **Session parameters** although it does show in the example. Might be good to add this as a parameter under `info mmap`?

Reply:
Updated for both suggestions.